packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1033,7 +1044,7 @@ mixin TextSelectionDelegate {
   /// * [EditableTextState.userUpdateTextEditingValue]: an implementation that
   ///   applies additional pre-processing to the specified [value], before
   ///   updating the text editing state.
-  void userUpdateTextEditingValue(TextEditingValue value, SelectionChangedCause cause);
+  void userUpdateTextEditingValue(TextEditingValue value, SelectionChangedCause cause, {TextSelectionHandleSide handleToView = TextSelectionHandleSide.extentHandle});
```

Comment:
Can you document handleToView in the doc comments here?  And if there's anywhere else that it is exposed publicly.

Reply:
I removed all `handleToView`.